src/localization/bundles/en.json

```diff
@@ -139,7 +139,8 @@
     "validate.config.files.heading": "Review configuration files",
     "validate.config.files.validate": "Validate",
     "validate.config.files.deploy": "Deploy",
-    "validate.config.files.msg.info0": "Validated model changes will be committed but are not automatically deployed. The model may be edited here in support of server add/replace use cases, or as needed to support service configuration changes. The deployment update process should be completed on the servers or configuration pages respectively.",
+    "validate.config.files.msg.warning": "Modifying the model directly may result in the cloud being in a bad state.",
```

Comment:
I would use different language here. "Modifying the model directly is an advanced use case and should be done with caution as incorrect changes may result in the cloud being in a bad state"